### PR TITLE
Make more effort to get title on xform upload

### PR DESCRIPTION
**Pull request by alxndrsn**
Current implementation works for me in Chrome 44, but not in Chrome 60.

### CODE REVIEW

**Comment by garethbowen** [comment]
The code looks fine, but I can't work out why the `$xml.find` would fail and the regex would succeed?

File: static/js/controllers/configuration-forms-xml.js [hunk 1]
```diff
@@ -59,7 +59,14 @@ angular.module('inboxControllers').controller('ConfigurationFormsXmlCtrl',
           var meta = results[1];
 
           var $xml = $($.parseXML(xml));
+
           var title = $xml.find('title').text();
+          if (!title) {
```
Comment (garethbowen) [inline static/js/controllers/configuration-forms-xml.js]: Maybe we should only use the regex version if the `$xml.find` doesn't work in modern Chrome? Do we know why it doesn't work?

Comment (alxndrsn) [reply static/js/controllers/configuration-forms-xml.js]: I think it's to do with XML namespaces.  But yes, it would be nice to understand _why_!

**Comment by alxndrsn** [comment]
I've added a comment with the information we know at the moment.  If we can get this merged now that would be really helpful for usability, and I will investigate the cause and potential fixes later if that's reasonable.

**Pull request merged**